README.md

```diff
@@ -2,44 +2,33 @@
+- Register access should be unsafe because we consider it as C FFI and many times theres HW undefined behavior that can be solved
   only at driver level. (e.g. sometimes also the order of writing register bitfields is important.
   discussion on this topic available here <https://github.com/rust-embedded/svd2rust/issues/714>).
 - No ownership because owned registers are an obstacle to writing low level drivers (LLD). Anyway writing
   LLDs requires a lot of unsafe code and ownership makes it more complex to access registers from interrupts
   and other threads. LLDs shall present safe APIs because only they can implement all logic for a safe usage of peripherals.
+  Moreover for many peripherals the splitting of peripheral is smaller unit is not obvious and depends on use cases.
 - Support [tracing](#tracing) of register accesses and additionally mocking of registers on non-embedded devices through
   external libraries. This allows the execution unit tests for code that uses the generated libraries on non-embedded devices.
-- PAC shall have ideally 0 dependencies to any other crates.
-
-##  chiptool features implemented by `svd2pac`
-
-- No owned data structure -> unsafe api that can be used from any interrupts or thread.
-- Register values are represented by structs. No write to register when updating a bitfield.
-- Enumerated values are represented with a struct + associated constants.
-- Essentially same API to access registers that chiptool uses.
-- Logging of register accesses.
-
-## Major differences to chiptool
-
-- Support for modify atomic assembly instruction (only Aurix).
-- Bitfields are structs with setters and getters. This prevents naming conflicts with bitfield names like `set_xxx`.
-- Bitfield setters consume the register struct and support a fluent api. Lower coding in closure.
-- Support for the "Cluster" tag in SVD files.
-- Support also for 8,16,64 bit size registers.
-- Avoid long type and bitfield names (chiptool concatenates identifiers) by generating modules for registers and svd clusters.
-- No support for SVD transformation.
-
-## Limitations
+- No macros. Absence of macros make easier the debugging.
+- PAC shall have 0 dependencies to any other crates.
+   - Exception: `--target=cortex-m`. In this case the generated PAC has some dependencies in order to be usable in ARM Cortex Rust ecosystem.
+- Use associated constant instead of `Enum` for bitfield values so user can easily create new values.
+  Enumeration constrains the possible value of a bitfield but many times the SVD
+  enum description has missing enumeration values. There are multiple reasons:
+  * Too many values for documentation/SVD.
+  * Valid values depends on other register values or condition so user documentation writer could decided to not describe in table.
+  * Lazyness of user manual writer. (Sorry but it sometimes happens ;-))
+
```

Comment:
```suggestion
- Use associated constants instead of `Enum` for bitfield values so users can easily create new values.
  Enumerations constrain the possible values of a bitfield but many times the SVD
  enum description has missing enumeration values. There are multiple reasons:
  * Too many values for documentation/SVD.
  * Valid values depend on other register values or conditions so documentation writers could decided to not list them in the SVD.
  * Lazyness of user manual writer. (Sorry but it sometimes happens ;-))

```